### DISCUSSION TITLE
Welcome to Tradovate-Python-Client Discussions!

### DISCUSSION THREAD

**Discussion by cullen-b** [cullen-b/Tradovate-Python-Client#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by virmanir** [comment]
Thanks for putting together this Tradovate library. I'm just getting started with the development of the integration between TradingView and Tradovate using AWS Lambda and python Chalice framework. Glad to help and get help like this library to save me time.
